MAUI/MauiAppBasic/Platforms/Tizen/Main.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+
+namespace MauiAppBasic
+{
+    internal class Program : MauiApplication

```

Comment:
Does Tizen work?

Reply:
Added code that throws PlatformNotSupportedException

Reply:
I would remove Tizen and MacCatalyst folders since they're not implemented, to keep the sample cleaner. Unless these folders are required for MAUI to compile? In that case, add a comment that clarifies that MAUI supports these platforms, but the sample just doesn't implement them.